crown-court-proceeding/build.gradle

```diff
@@ -30,17 +30,25 @@ java {
     targetCompatibility = JavaVersion.VERSION_17
 }
 
+sourceSets {
+    testSqsIntegration {
+        compileClasspath += sourceSets.main.output
```

Comment:
Could you please rename 'testSqsIntegration' to 'integration' and integration to sqs to make it easy to understand?. It also requires changes to sourceSets?

Reply:
Hi @skasthuri36326 Like I mentioned in our discussion, I named it as such intentionally so that the test packages come after the main. If I rename it as `integration`, it falls before the main package. So solely for the purpose of common package structure I added the test prefix.